tools/homarr-dev/internal/cli/manage.go

```
@@ -0,0 +1,329 @@
+	cmd.Flags().BoolVarP(&watch, "watch", "w", false, "Watch CI checks until completion")
+	return cmd
+}
+
+func newTable(headers ...string) *tabwriter.Writer {
+	writer := tabwriter.NewWriter(os.Stdout, 0, 2, 2, ' ', 0)
+	row(writer, headers...)
+	return writer
+}
+
+func row(writer *tabwriter.Writer, cells ...string) {
+	fmt.Fprintln(writer, strings.Join(cells, "\t"))
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintln` is not checked

(errcheck)